src/views/NavigationContext.js

```diff
@@ -1,5 +1,5 @@
-import createReactContext from 'create-react-context';
+import React from 'react';
 
-const NavigationContext = createReactContext(undefined);
+const NavigationContext = React.createContext(null);
```

Comment:
Doesn't `undefined` work? Changing the initial value might break some code

Reply:
All the other examples of removing this library in GitHub were using `null`, so I matched that since explicit assingment/passing of `undefined` is kind of considered an anti-pattern.

Looking at React sources, they mostly pass `null` in this scenario:
https://github.com/facebook/react/search?p=3&q=createContext&unscoped_q=createContext
https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/react-test-renderer/src/__tests__/ReactTestRendererTraversal-test.js#L24

Reply:
I was talking about the usage of the library. If someone is checking for context to be `undefined` and we change it to `null` it'll be a breaking change. Might not matter, but just in case

Reply:
Changed it back to undefined and added a TODO comment.

Reply:
Seems you forgot to push the changes

Reply:
when change is pushed i can land :)